packages/commonjs/README.md

```diff
@@ -106,6 +106,13 @@ Default: `true`
 
 If false, skips source map generation for CommonJS modules.
 
+### `transformMixedEsModules`
+
+Type: `Boolean`<br>
+Default: `false`
+
+In case of modules that contain commonjs code you can enable/disable the transformer. Set it to `true` if you know that `require` calls should be transformed, or `false` if you know that the code contains env detection and the `require` should survive the transform.
```

Comment:
```suggestion
Instructs the plugin whether or not to enable mixed module transformations. This is useful in scenarios with mixed ES and CommonJS modules. Set to `true` if it's known that `require` calls should be transformed, or `false` if the code contains env detection and the `require` should survive a transformation.
```

It would be useful for users to also link to an explanation of what a mixed module is, as well as information on what the transformation looks like. This won't be common knowledge to most folks, and we should try to make it discoverable.

Reply:
Nice, I also just added a change suggestion 😝 I guess you can mix an match now @danielgindi